projects/templates/src/lib/components/po-page-dynamic-table/po-page-dynamic-table.component.ts

```diff
@@ -1026,6 +1028,11 @@ export class PoPageDynamicTableComponent extends PoPageDynamicListBaseComponent
   }
 
   private updateTableActions() {
-    this.tableActions = [...this._defaultTableActions, ...this._customTableActions];
+    const index = this._defaultTableActions.length - 1;
```

Comment:
Caso estiver em outro idiomas (PO_USER_LOCALE: en  ou idioma do browser) ocorre um erro, pois o this._defaultTableRemoveAction não existe.

Pode testar isso alterando o idioma do browser ou adicionar PO_USER_LOCALE com en no storage do browser
 ..


Acredito que podemos fazer um pouco mais simples/legível até mesmo para manutenção futura do código e mais segura, não fixando apenas na ultima posição, ex:
```
  private updateTableActions() {
    const tableActionDelete = this._defaultTableActions.find(tableAction => tableAction.label === this.literals.tableActionDelete);
    const defaultTableActionsWithoutActionDelete =
      this._defaultTableActions.filter(tableAction => tableAction.label !== this.literals.tableActionDelete);

    this.tableActions = [...defaultTableActionsWithoutActionDelete, ...this._customTableActions, tableActionDelete];
  }

```